src/test/java/uk/gov/ida/matchingserviceadapter/domain/VerifyMatchingServiceResponseTest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.ida.matchingserviceadapter.domain;
+
+import org.junit.Test;
+import uk.gov.ida.matchingserviceadapter.saml.transformers.outbound.OutboundResponseFromMatchingService;
+
+import static org.hamcrest.CoreMatchers.sameInstance;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class VerifyMatchingServiceResponseTest {
+    @Test
+    public void ctor() {
```

Comment:
Please rename this test to describe what behaviour it is verifying (or delete if we don't need it)